lib/galaxy/container/docker_swarm_manager.py

```diff
@@ -0,0 +1,560 @@
+"""
+Docker Swarm mode management
+"""
+from __future__ import absolute_import
+
+import argparse
+import errno
+import logging
+import os
+import subprocess
+import sys
+import time
+
+try:
+    import daemon
+    import daemon.pidfile
+    import lockfile
+except ImportError:
+    daemon = None
+
+try:
+    import galaxy   # noqa: F401 this is a test import
+except ImportError:
+    sys.path.insert(0, os.path.abspath(os.path.join(
+        os.path.dirname(__file__),
+        os.pardir,
+        os.pardir)))
+
+from galaxy.config import (
+    configure_logging,
+    find_path,
+    find_root,
+)
```

Comment:
Having given this a bit more thought... the only thing the Galaxy stuff is used for is locating the Galaxy config file so that `containers_conf.yml` can be located - except that the automatic loading in `galaxy.container.docker_swarm` doesn't use that option, it specifies the path to `containers_conf.yml` directly via the `--containers-conf-file` option. Outside of that, the swarm manager is useful on its own and apart from Galaxy.

tl;dr I can just remove the Galaxy stuff. Thoughts?